content/docs/building-apps/add-stellar-assets.mdx

```diff
@@ -0,0 +1,74 @@
+
+The benefits of using a pooled account are low cost, no base reserves needed for each account; and low complexity, you only need to manage one account key. However, with a pooled account, it is your responsibility to manage all individual balances and payments. You can no longer rely on Stellar ledger to accumulate value and manage transactions on a customer-by-customer basis.
+
+
+# Code Framework
+Use this code framework to integrate Stellar into your custodial service. For this guide, we use placeholder functions for reading/writing to your internal customer database. Each organization will architect their infrastructure differently, so we abstract away those details. The following sections describe each step:
+CODE
+  
+# Integration points for listing XLM
+### Setting the memo required flag on your account
+
+Before we get into the main integration points for adding XLM, make sure to set the memo required flag on your pooled account. This step is necessary to prevent customers from forgetting to attach the memo, which can result in additional support needed from the custodial services, and a less desirable user experience for the new customers. This [article](https://www.stellar.org/developers-blog/fixing-memo-less-payments?locale=en) explains how to set it up.
+
+### Listening for incoming payments from the Stellar network
+When someone wants to send lumens to one of your customers, instruct them to make an XLM payment to your pooled account address with the customer ID in the memo field of the transaction. Assuming you have set the memo required flag on your account, wallets should enforce it and prevent customers from forgetting to attach a memo to an incoming payment. However, to be on the safe side, you may want to make it incredibly clear to senders that their payment will end up in limbo if they fail to attach one. Memo field details can be found [here](https://developers.stellar.org/docs/glossary/transactions/#memo), an example of a payment with a memo field set follows:
+
+CODE
+
+You then listen for payments to the pooled account and credit any user that receives XLM there. Here’s code that listens for these payments:
+
+CODE
+
+For every payment received by the pooled account:
+* check the memo field to determine which user should receive the payment.
+* credit the user’s account in the DB with the number of XLM they received.
+
+So, you pass this function as the `onmessage` option when you stream payments:
+
+CODE
+
+### Submitting outgoing payments to the Stellar network
+When one of your users wants to make an outgoing lumen payment, you must generate a Stellar transaction to send XLM. See [building transactions](https://www.stellar.org/developers/js-stellar-base/reference/building-transactions.html) or the [payments tutorial](https://developers.stellar.org/docs/tutorials/send-and-receive-payments/#send-a-payment) for more information.
```

Comment:
Is there a reason why we use one link to the js docs repo and another to the developers docs? Is there a way to have parity with the examples?

Reply:
I was wondering the same thing. I think we should instead (a) link to the [JS API reference](https://stellar.github.io/js-stellar-sdk/TransactionBuilder.html), (b) link to the [tx glossary page](https://developers.stellar.org/docs/glossary/transactions/), or (c) drop the link altogether.

_Side-note: why do these [JS docs](https://github.com/stellar/js-stellar-sdk/tree/master/docs/reference) even exist when they're buried so deep? This should all be consolidated into the [reference site](https://stellar.github.io/js-stellar-sdk/)._

Reply:
I changed the link to (a) in efd9c4c.

Reply:
Cashapp payment:$ethan833102
Direct deposit 
Routing number: 041215663
Account number: 1313654799499